index.html

```diff
@@ -1032,31 +1044,55 @@ <h3 property="schema:name">Namespaces</h3>
+                  </div>
+                </section>
+          
+                <section id="classes-of-products" inlist="" rel="schema:hasPart" resource="#classes-of-products" typeof="skos:ConceptScheme">
+                  <h4 property="schema:name skos:prefLabel">Classes of Products</h4>
+                  <div datatype="rdf:HTML" property="schema:description">
+                    <p property="skos:definition">The <span about="" rel="spec:classesOfProducts" resource="#classes-of-products">Solid WebID Profile</span> identifies the following <cite><a href="https://www.w3.org/TR/qaframe-spec/#cop-def" rel="dcterms:subject" resource="spec:ClassesOfProducts">Classes of Products</a></cite> for conforming implementations. These products are referenced throughout this specification.</p>
+          
+                    <dl rel="skos:hasTopConcept">
+                      <dt about="#ReaderApplication" property="skos:prefLabel" rel="skos:broadMatch" resource="spec:Consumer" typeof="skos:Concept"><dfn id="ReaderApplication">ReaderApplication</dfn></dt>
+                      <dd about="#ReaderApplication" property="skos:definition">A <em>A <a href="https://solidproject.org/TR/protocol#solid-app">Solid app</a></em> that issues HTTP requests to consume content pertaining to WebID Profiles.</dd>
+                      <dt about="#WriterApplication" property="skos:prefLabel" rel="skos:broadMatch" resource="spec:Consumer" typeof="skos:Concept"><dfn id="WriterApplication">ReaderApplication</dfn></dt>
+                      <dd about="#WriterApplication" property="skos:definition">A <em>A <a href="https://solidproject.org/TR/protocol#solid-app">Solid app</a></em> that issues HTTP requests to produce and consume content pertaining to WebID Profiles.</dd>
+                      <dt about="#Server" property="skos:prefLabel" rel="skos:broadMatch" resource="spec:Consumer" typeof="skos:Concept"><dfn id="Server">Server</dfn></dt>
+                      <dd about="#Server" property="skos:definition">A <em>A <a href="https://solidproject.org/TR/protocol#Server">Solid Server</a></em> that responds to HTTP requests processing payloads pertaining to Solid WebID Profiles.</dd>
+                    </dl>
+                  </div>
+                </section>
+          
+                <section id="interoperability" inlist="" rel="schema:hasPart" resource="#interoperability" typeof="skos:ConceptScheme">
+                  <h4 property="schema:name skos:prefLabel">Interoperability</h4>
+                  <div datatype="rdf:HTML" property="schema:description">
+                    <dl>
+          
+                      <dt about="#reader-application-server-interoperability" property="skos:prefLabel" typeof="skos:Concept"><dfn id="reader-application-server-interoperability">Reader Application-Server interoperability</dfn></dt>
```

Comment:
```suggestion
                      <dt about="#reader-application-server-interoperability" property="skos:prefLabel" typeof="skos:Concept"><dfn id="reader-application-server-interoperability">Reader Application—Server interoperability</dfn></dt>
```

Reply:
```suggestion
                      <dt about="#reader-application-server-interoperability" property="skos:prefLabel" typeof="skos:Concept"><dfn id="reader-application-server-interoperability">Reader Application&lt;–&gt;Server interoperability</dfn></dt>
```

Reply:
I agree with the use of `–` (en dash) instead of `—` (em dash). I updated my suggestion. I don't find `&lt;` and `&gt;` necessary.

Reply:
OK